tests/yoti_common/converters/attribute.converter.spec.js

```diff
@@ -239,6 +239,10 @@ describe('AttributeConverter', () => {
         expect(errMessage).toBe('Warning: value is NULL', `Content Type: ${contentType}`);
       });
     });
+    it('should return Date values', () => {
+      const converted = AttributeConverter.convertValueBasedOnContentType('1979-01-01', CONTENT_TYPE_DATE);
+      expect(converted).toBeInstanceOf(Date);
```

Comment:
Probably good to check the date is matching ;-)
Please add a few lines to check the yeah, month...